6.traits.md

```diff
@@ -53,7 +53,7 @@ Implementations of an OAM runtime SHOULD make all supported traits discoverable
 
```

Comment:
We no longer have workload `type` anymore.  I think we need to modify the spec temporarily until we introduce `type` back.

Reply:
The good thing is that we have only 1 workload now so it still works :) 
I think if we can quickly converge on the idea that traits can be applied to a certain "policy" then we can at least put that idea in the spec before the Kubecon demo.

Reply:
> Yeah there is a glaring hole here right now.

The hole was there (albeit arguable smaller) already as we didn't define any workload "type" outside of core workloads. One can't find any traits that apply to an extended workload.

Reply:
> don't feel this would affect the spec or implementation that much, i.e.:

In this case, I wonder how many core workload Definitions are we going to put in the spec?

Reply:
btw, the demo only uses Kind: server